index.html

```diff
@@ -232,6 +232,10 @@ <h3 id="build"> Build </h3>
 </tbody></table>
 <h3 id="downloads"> Downloads </h3>
 <table class='badge'><tbody>
+  <tr><th data-keywords='sourcegraph' data-doc='sourcegraphDoc'> Sourcegraph (# references other repos): </th>
```

Comment:
It looks like you are not using documentation, so `data-doc='sourcegraphDoc` does not reference a DOM object with `id="sourcegraphDoc"`. Additionally, you do not need to put `data-keywords='sourcegraph'`, since that word is already in the text of the `th`.

Also, this badge is unrelated to downloads. Could you put it at the bottom of the miscellaneous section, please?

Finally, perhaps you can simplify the title to make it shorter? What matters most is that people can understand what it does; hopefully the badge is enough, so that the part in parentheses is not necessary.